src/lib/net.py

```diff
@@ -54,15 +54,17 @@ def download_snapshot(
     # The skip_existing flag is ignored if the file does not already exist
     if not skip_existing or not file_path.exists():
         with open(file_path, "wb") as file_handle:
-            download(url, file_handle)
+            download(url, file_handle, **download_opts)
 
     # Output the downloaded file path
     return str(file_path.absolute())
 
 
-def download(url: str, file_handle: BinaryIO, progress: bool = False) -> None:
+def download(
+    url: str, file_handle: BinaryIO, progress: bool = False, spoof_browser: bool = True
+) -> None:
     """ https://stackoverflow.com/a/37573701 """
-    headers = {"User-Agent": "Mozilla"}
+    headers = {"User-Agent": "Safari"} if spoof_browser else {}
```

Comment:
Is this change related to the move from github to cloud storage?  This is used in downloading the source data, correct?

Reply:
As of yesterday, the WorldBank data source stopped working and this is needed to fix it. It's all part of the same PR because it's high-priority but I can break it up if you prefer.

Reply:
Either way is fine, but since it's high-priority I suggest you break it up so you can get it in before we're done reviewing the rest of the PR.

Reply:
At this point, the data is no longer being updated because of the GitHub limits. So fixing it separately does not yield results any faster. Other than the renaming, this PR is not all that controversial so I don't expect for reviewing to take very long. I'm mainly looking for any potential mistakes.

The data is on the GCS bucket and I updated [the explorer to use it](https://open-covid-19.github.io/explorer). Everything looks good to me, so I'm fairly confident this CL is doing the right thing.

Reply:
Looks good, the one thing I would suggest if you haven't already done it is to run the old and new versions of publish.py on the same inputs and compare their outputs.  As I understand it, the only changes to publish.py are renaming of "master" to "main" and separating out the v1 stuff into the separate backcompat script, correct?